_posts/en/newsletters/2018-10-30-newsletter.md

```diff
@@ -0,0 +1,246 @@
+---
+title: "Bitcoin Optech Newsletter #19"
+permalink: /en/newsletters/2018/10/30/
+name: 2018-10-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter suggests an update for C-Lightning users,
+describes a discussion about BIP69 deterministic input/output ordering
+on the mailing list, notes public overt ASICBoost support is available
+for miners using Antminer S9, provides links to information about
+Square's open sourced Subzero HSM-based multisig cold storage solution
+and summarizes the recent Lightning Network Residency and Hackday in New York City.
+Also included are selected recent Q&A from Bitcoin StackExchange and
+descriptions of notable code changes in popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+- **Update to [C-Lightning 0.6.2][]:** fixes a bug where the node would
+  send an excessive number of update announcements to its peers about
+  dead channels.
+
+## News
+
+- **[BIP69][] discussion:** this BIP from 2015 adopted by several notable
+  wallets specifies an optional method for deterministically ordering
+  inputs and outputs within a transaction based on the public contents
+  of the transaction.  However, other wallets have not adopted it (or
+  even rejected it as unsuitable for adoption), leading perhaps to a
+  "worst of both worlds" situation where wallets using BIP69 can be
+  fairly easily identified and so wallets not using BIP69 may also be
+  easier to identify by negation.
+  
+    In this [thread][bip69 thread] to the Bitcoin-Dev mailing list, Ryan
+    Havar suggests that one reason wallet authors like BIP69 is that its
+    deterministic ordering makes it easy and fast for their tests to
+    ensure that they haven't leaked any information about the source of
+    their inputs or the destination of their outputs (e.g.  in some old
+    wallets, the fist output always went to the recipient and the second
+    output was always change---making coin tracking trivial).  Havar
+    then suggests an alternative deterministic ordering based on private
+    information that would be available to the test suite but not
+    exposed by production wallets, allowing developers who want to
+    thwart block chain analysis---but also have simple and fast
+    tests---to migrate away from BIP69.
+
+- **Overt ASICBoost support for S9 miners:** support for this
+  efficiency-improving feature was announced by both [Bitmain][bitmain oab]
+  and [Braiins][braiins oab] this week.  Both covert and overt ASICBoost
```

Comment:
I think this sentence could be clearer. I've made an attempt in my commit, but I'm not sure if it's any better. Let me know what you think.

We could also link directly to one of the following:
the ASICboost paper : http://www.math.rwth-aachen.de/~Timo.Hanke/AsicBoostWhitepaperrev5.pdf
stack exchange answer: https://bitcoin.stackexchange.com/questions/43579/how-does-asicboost-work
blog post by ASICboost author: https://bitslog.wordpress.com/2017/04/10/the-relation-between-segwit-and-asicboost-covert-and-overt/